packages/components/src/font-size-picker/styles.ts

```diff
@@ -44,12 +43,3 @@ export const Controls = styled.div< {
 	${ ( props ) =>
 		! props.__nextHasNoMarginBottom && `margin-bottom: ${ space( 6 ) };` }
 `;
-
-export const ResetButton = styled( Button )< {
-	size: FontSizePickerProps[ 'size' ];
-} >`
-	&&& {
-		height: ${ ( props ) =>
-			props.size === '__unstable-large' ? '40px' : '30px' };
-	}
```

Comment:
Not sure when it regressed, but these height overrides weren't currently working. The reset button was rendered as 24px no matter what.

In this PR, I went ahead and simplified it a bit so that the reset button is 40px in the `__unstable-large` case, and 24px otherwise. This should be ok because we'll be phasing out the non-large variants anyway.

Reply:
I definitely considered it, but a 2px difference between the text field and the button would irritate a designer more than a 6px difference would 😆